week2/materials/homework/assignments.md

```diff
@@ -4,38 +4,39 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `https://s3.console.aws.amazon.com/s3/buckets/g3-week2-tithi?region=us-east-1&tab=objects` 
+website url: `https://s3.console.aws.amazon.com/s3/buckets/g3-week2-tithi?region=us-east-1&tab=objects`
```

Comment:
This seems to be the wrong url, this is linking to the bucket not a public available url to a website

Reply:
So sorry. I have yet again added the wrong one. Should be this one instead `https://g3-week2-tithi.s3.amazonaws.com/index.html`. Will update it on the .md file today :)